content/user/ui/settings/Integrations.md

```diff
@@ -80,11 +83,11 @@ You are connected!
 
 ## Bugtrackers
 
-TeamServer can connect to bugtrackers in order to streamline the remediation process. Individual or groups of discovered vulnerabilities can be sent directly to a configured bugtracker so that development groups may implement a fix.
+Contrast can connect to bugtrackers in order to streamline the remediation process. Individual or groups of discovered vulnerabilities can be sent directly to a configured bugtracker so that development groups may implement a fix.
```

Comment:
@kaitlynmortimer Can you incorporate the info from this note in the ticket: "It would be nice if we could also address that we now track what's being sent to bugtrackers in the general section of bugtrackers somehow. Users can filter by what's being tracked/not being tracked and we present an indicator in the main Vuln grid. Those implementation ticket/notes are in
CONTRAST-11058
CONTRAST-11059"